agent/engine/daemonmanager/daemon_manager_test.go

```diff
@@ -0,0 +1,269 @@
+//go:build unit
+// +build unit
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package daemonmanager
+
+import (
+	"encoding/json"
+	"fmt"
+	apitask "github.com/aws/amazon-ecs-agent/agent/api/task"
```

Comment:
Format the imports please-

```
Standard Import

Same Package

AWS/Internal packages

External packages
```

Reply:
I thought my IDE was smart. I will fix this.